src/DevTools/PerformanceComparison/Program.cs

```diff
@@ -34,59 +34,59 @@ function move(n, src, dst, via)
 					move(n - 1, via, dst, src)
 				end
 			end
- 
+
 			for i = 1, 10000 do
 				move(4, 1, 2, 3)
 			end
 			";
 		static  string scriptText22 = @"
 N = 8
- 
+
 board = {}
 for i = 1, N do
-    board[i] = {}
-    for j = 1, N do
+	board[i] = {}
+	for j = 1, N do
 		board[i][j] = false
-    end
+	end
 end
- 
+
 function Allowed( x, y )
-    for i = 1, x-1 do
-	if ( board[i][y] ) or ( i <= y and board[x-i][y-i] ) or ( y+i <= N and board[x-i][y+i] ) then 
-  	    return false 
+	for i = 1, x-1 do
+	if ( board[i][y] ) or ( i <= y and board[x-i][y-i] ) or ( y+i <= N and board[x-i][y+i] ) then
+		return false
+	end
 	end
-    end		
-    return true
+	return true
 end
- 
+
 function Find_Solution( x )
-    for y = 1, N do
-	if Allowed( x, y ) then 
-  	    board[x][y] = true 
-	    if x == N or Find_Solution( x+1 ) then
+	for y = 1, N do
```

Comment:
Looks like the existing code was a mix of tabs and spaces and my IDE tried to make it consistent, but messed up the indentation levels.

I'll sort this out.